plugins/pom.xml

```diff
@@ -39,6 +39,7 @@
         <module>past-event</module>
         <module>tracked-event</module>
         <module>kafka-injector</module>
+        <module>tracardi-plugins</module>
```

Comment:
I would prefer we do not add a "client" specific plugin here. Rather these actions, if we keep them (see below), should become part of the base plugin.